config/samples/saas_v1alpha1_system.yaml

```diff
@@ -0,0 +1,191 @@
+        - web_hooks.yml
+        - zync.yml
+    # TODO: can be removed?
+    seed:
+      masterAccessToken:
+        fromVault:
+          path: secret/data/path/system-seed
+          key: MASTER_ACCESS_TOKEN
+      masterDomain: multitenant-admin
+      masterUser:
+        fromVault:
+          path: secret/data/path/system-seed
+          key: MASTER_USER
+      masterPassword:
+        fromVault:
+          path: secret/data/path/system-seed
+          key: MASTER_PASSWORD
+      adminAccessToken:
+        fromVault:
+          path: secret/data/path/system-seed
+          key: ADMIN_ACCESS_TOKEN
+      adminUser:
+        fromVault:
+          path: secret/data/path/system-seed
+          key: ADMIN_USER
+      adminPassword:
+        fromVault:
+          path: secret/data/path/system-seed
+          key: ADMIN_PASSWORD
+      adminEmail: admin@example.com
+      tenantName: example
+    databaseDSN:
+      fromVault:
+        path: secret/data/path/system-database
+        key: URL
+    databaseSecret:
+      fromVault:
+        path: secret/data/path/system-app
+        key: DB_SECRET
+    eventsSharedSecret:
+      fromVault:
+        path: secret/data/path/system-app
+        key: EVENTS_SHARED_SECRET
+    secretKeyBase:
+      fromVault:
+        path: secret/data/path/system-app
+        key: SECRET_KEY_BASE
+    recaptcha:
+      publicKey:
+        fromVault:
+          path: secret/data/path/system-recaptcha
+          key: RECAPTCHA_PUBLIC_KEY
+      privateKey:
+        fromVault:
+          path: secret/data/path/system-recaptcha
+          key: RECAPTCHA_PRIVATE_KEY
+    accessCode:
+      fromVault:
+        path: secret/data/path/system-app
+        key: ACCESS_CODE
+    # TODO: is this used?
+    segment:
+      deletionWorkspace: example
+      deletionToken:
+        fromVault:
+          path: secret/data/path/system-app
+          key: SEGMENT_DELETION_TOKEN
+      writeKey:
+        fromVault:
+          path: secret/data/path/system-app
+          key: SEGMENT_WRITE_KEY
+    github:
+      clientID:
+        fromVault:
+          path: secret/data/path/system-app
+          key: GITHUB_CLIENT_ID
+      clientSecret:
+        fromVault:
+          path: secret/data/path/system-app
+          key: GITHUB_CLIENT_SECRET
+    # TODO: can be removed?
+    metrics:
+      user:
+        fromVault:
+          path: secret/data/path/system-app
+          key: PROMETHEUS_USER
+      password:
+        fromVault:
+          path: secret/data/path/system-app
+          key: PROMETHEUS_PASSWORD
+    redhatCustomerPortal:
+      clientID:
+        fromVault:
+          path: secret/data/path/system-app
+          key: RH_CUSTOMER_PORTAL_CLIENT_ID
+      clientSecret:
+        fromVault:
+          path: secret/data/path/system-app
+          key: RH_CUSTOMER_PORTAL_CLIENT_SECRET
+    bugsnag:
+      apiKey:
+        fromVault:
+          path: secret/data/path/system-app
+          key: BUGSNAG_API_KEY
+    memcachedServers: ""
```

Comment:
can this var be emtpy?

Reply:
It seems that it can. This is how system was deployed in dev.

Reply:
ok!